index.html

```diff
@@ -0,0 +1,28 @@
+
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
```

Comment:
Good job creating your first website. You have done a great job so far, however I suggest that you add a viewport meta tag within the head element to make your website accessible on all devices.